org.sonarlint.eclipse.ui/src/org/sonarlint/eclipse/ui/internal/properties/RulesConfigurationPart.java

```diff
@@ -201,7 +201,7 @@ private void createTreeViewer(Composite parent) {
   private void createRuleDescriptionPart(Composite parent) {
     try {
       Browser browser = new Browser(parent, SWT.FILL);
-      browser.setText("(No rule selected.)");
+      browser.setText("<small><em>(No rule selected)</em></small>");
```

Comment:
Is it not supposed to be "no rule**s**" ?

Reply:
In terms of grammar, I think both singular and plural would be correct, but mean slightly different things. Plural would imply that a multi-selection is possible, but in fact it won't make a difference for rule display, since only one rule will be displayed at a time. I think singular will be less confusing.

Reply:
OK, I trust you :)